.github/workflows/helmchart-ocm.yaml

```diff
@@ -94,9 +94,17 @@ on:
           passed as `ctx`-input to `merge-ocm-fragments` action.
         required: false
         type: string
+      github-environment:
+        required: false
+        default: ""
+        type: string
```

Comment:
GHA-workflow-inputs do feature a type `environment`. I assume this should be used.

Reply:
I double-checked. Differently than [documented](https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#onworkflow_dispatchinputsinput_idtype), it does not seem to be allowed to specify `environment` as input-type. 🤦🏻‍♂️